web/web/api/v1/sns_read.py

```diff
@@ -0,0 +1,30 @@
+from flask import Flask, request, Blueprint
+import json
+
+sns_read_api_bp =  Blueprint('sns_read_api_bp', __name__)
+
+
+@sns_read_api_bp.route('/sns_read', methods = ['GET', 'POST'])
```

Comment:
does AWS really POST and GET to this endpoint? Seems weird to support both

Reply:
You're right, converted to post only